backend/src/lib/constants/env-vars-schema.js

```diff
@@ -0,0 +1,12 @@
+export default {
+  CLIENT_ID: /.+/,
+  CLIENT_ID_PRODUCTION: /.+/,
+  LOCAL_STORAGE_PATH: /.+/,
+  MONGODB_URL: /^mongodb:\/\/.+/,
+  NOW_PLAYING_URL: /^https?:\/\/.+/,
+  SPOTIFY_ID: /^[0-9a-f]{32}$/,
+  SPOTIFY_NEW_TRACKS_ADDED_LIMIT: /^\d+$/,
+  SPOTIFY_SECRET: /^[0-9a-f]{32}$/,
+  WS_MOPIDY_PORT: /^\d+$/,
+  WS_MOPIDY_URL: /.+/
```

Comment:
I'd appreciate any suggestions you have for strengthening these regexes, especially the `/.+/` ones :slightly_smiling_face:

Reply:
It's an interesting concept and not one I've seen before. It does introduce an extra overhead in managing constants as we now have them in 2 locations:
https://github.com/kyan/jukebox-js/blob/0482568e345c58bbe557e1218953338ec87f5e84/backend/src/lib/constants/env-vars-schema.js
https://github.com/kyan/jukebox-js/blob/master/docker-compose.yml#L38-L49
so we can see how that goes. Although I'm not sure how many extra constants we'll need to add TBH.

Also the build has broken so needs fixing and have you run this locally to see if there are any issue?